src/os/ObjectStore.h

```diff
@@ -700,6 +700,9 @@ class ObjectStore {
 
   /// OMAP
   /// Get omap contents
+  virtual void fix_omap() {}
+  virtual void set_omap_legacy(bool legacy) {}
```

Comment:
Instead of extending ObjectStore interface with set_/get_omap_legacy() methods you might consider doing that specifically at BlueStore and use dynamic cast in store_test.cc which is the only user of them